lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -37,43 +34,6 @@ def _tick_only(ax, bottom_on, left_on):
     ax.axis["left"].toggle(ticklabels=left_off, label=left_off)
 
 
-class Colorbar(mcolorbar.Colorbar):
-    def _config_axes_deprecated(self, X, Y):
```

Comment:
This is a subclass of `mpl_toolkits.axes_grid1.colorbar.Colorbar` (not `matplotlib.Colorbar`) that adds a single private method that is not even used anywhere else.